posthog/models/organization.py

```diff
@@ -186,6 +186,7 @@ class DefaultExperimentStatsMethod(models.TextChoices):
     objects: OrganizationManager = OrganizationManager()
 
     is_platform = models.BooleanField(default=False, null=True, blank=True)
+    is_env_rollback_triggered = models.BooleanField(null=True, blank=True)
```

Comment:
Hmm I don't like adding a tmp col because it's hard to drop it. Sorry I know it's annoying to change this. 

Could we instead use redis? There aren't that may so it should be possible to store in an array in redis.